### PR TITLE
Added Routing of all pages

### PR DESCRIPTION
**Pull request by mehulsatardekar**
Files To check 
   -- Routing.jsx
   -- App.js

### CODE REVIEW

File: src/components/Routes/Routings.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    <CartData>
+        <Routes>
+          <Route path="/" element={<Home />} />
+           <Route path="/login" element={ !isAuth ? <Login/> : <Navigate to= "/"/> }/>
+          <Route path="/restaurants" element={<Restaurants />} />
+          <Route path="/restauranthome/:prodid" element={<RestaurantHome />} />
+          <Route path="/wishlist" element={<WishList />} />
+          <Route path="/cart" element={<Cart />} />
+          <Route path="/mockman" element={<Mockman />} />
+           <Route path="*" element={<NotFound/>}/>
```
Comment (rahulyadav139) [inline src/components/Routes/Routings.jsx:28]: Great you have a protected login page with respective auth conditions. Great work.

**Review by rahulyadav139 — APPROVED** [review]

File: src/components/Routes/Routings.jsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import { CartData } from "../../contexts/Cart/CartContext";
+import { authContext  } from "../../contexts/AuthContext/AuthContext";
+
+import { useContext } from "react";
+const Routings = () => {
+
+  const {auth:{isAuth}} = useContext(authContext);
+
+  return (
+    <CartData>
```
Comment (manish-GH) [inline src/components/Routes/Routings.jsx:19]: Recommend: Shift this context to App.js

File: src/components/Routes/Routings.jsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+import Cart from "../UI/Cart/Home/Home";
+import Login from "../UI/Login/Login";
+import NotFound from "../UI/NotFound/NotFound" 
+import { CartData } from "../../contexts/Cart/CartContext";
+import { authContext  } from "../../contexts/AuthContext/AuthContext";
+
+import { useContext } from "react";
+const Routings = () => {
+
+  const {auth:{isAuth}} = useContext(authContext);
```
Comment (manish-GH) [inline src/components/Routes/Routings.jsx:16]: Recommend: `useContext(authContext)` could have been exported from the AuthContext as `useAuthContext`

File: src/components/Routes/Routings.jsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
+import {  Routes, Route , Navigate} from "react-router-dom";
+import Mockman from "mockman-js";
+import Home from "../UI/HomePage/Home/Home";
+import Restaurants from "../UI/RestaurantsPage/Restaurants/Restaurants";
+import RestaurantHome from "../UI/RestaurantsHome/RestaurantHome/RestaurantHome";
+import WishList from "../UI/WishList/WishListHome/WishListHome";
+import Cart from "../UI/Cart/Home/Home";
+import Login from "../UI/Login/Login";
```
Comment (yaswanthmyneni) [inline src/components/Routes/Routings.jsx:8]: suggestion: create index file in UI folder and export all UI from it

File: src/components/Routes/Routings.jsx [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+import NotFound from "../UI/NotFound/NotFound" 
+import { CartData } from "../../contexts/Cart/CartContext";
+import { authContext  } from "../../contexts/AuthContext/AuthContext";
```
Comment (yaswanthmyneni) [inline src/components/Routes/Routings.jsx:11]: same for contexts

**Review by manish-GH — APPROVED** [review]
Nice work on handling the routes. Consider these minor changes and feel free to merge

**Review by yaswanthmyneni — APPROVED** [review]
code is good, you can merge it.

**Comment by mehulsatardekar** [comment]
@manish-GH @yaswanthmyneni yep agree that I haven't used the barrel exporting basically I have done this thing in my Video library project and will implement these changes here also once I merge the all pr. thanks guys for suggestions.

**Pull request merged**